lib/ui/painting/matrix.cc

```diff
@@ -32,6 +32,23 @@ SkMatrix ToSkMatrix(const tonic::Float64List& matrix4) {
   return sk_matrix;
 }
 
+SkM44 ToSkM44(const tonic::Float64List& matrix4) {
+  FML_DCHECK(matrix4.data());
+  SkM44 sk_m44;
+  for (int i = 0; i < 16; ++i) {
+    sk_m44.setRC(i % 4, i / 4, SafeNarrow(matrix4[i]));
```

Comment:
Do we want to assume the compiler is optimizing this nicely for us? These are potentially a lot of instructions to save a few lines of code.